lib/abci/handlers/query/identityIdsByPublicKeyHashesQueryHandlerFactory.js

```diff
@@ -46,8 +54,19 @@ function identityIdsByPublicKeyHashesQueryHandlerFactory(
       }),
     );
 
+    const value = {
+      data: identityIds,
+    };
+
+    if (includeProof) {
+      value.proof = rootTree.getFullProof(
+        publicKeyToIdentityIdStoreRootTreeLeaf,
+        [publicKeyHashes],
```

Comment:
`publicKeyHashes` is already an array of keys, so you don't need to wrap it into an array.